### PR TITLE
Add descriptive title element and breadcrumbs

### PR DESCRIPTION
**Pull request by eagletmt**
@cookpad/dev-infra please review

### CODE REVIEW

File: app/views/barbeque/job_definitions/new.html.haml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+- content_for(:title, 'New job definition - Barbeque')
+- content_for(:header) do
+  %ol.breadcrumb
+    %li= link_to('Home', root_path)
+    %li= link_to(@job_definition.app.name, app_path(@job_definition.app.id))
```
Comment (eisuke) [inline app/views/barbeque/job_definitions/new.html.haml]: Just initialized `@job_definition` has not `app` ?

File: app/views/barbeque/job_definitions/index.html.haml [hunk 2]
```diff
@@ -1,3 +1,8 @@
+- content_for(:title, 'Job definitions - Barbeque')
+- content_for(:header) do
+  %ol.breadcrumb
+    %li= link_to('Home', root_path)
```
Comment (hogelog) [inline app/views/barbeque/job_definitions/index.html.haml]: Forgot `%li.active` item?

File: app/views/barbeque/job_definitions/new.html.haml (hunk 1 above)
Comment (eagletmt) [inline app/views/barbeque/job_definitions/new.html.haml]: `@job_definition.app` exists when user comes from "New definition" button in /apps/:app_id (I think this is normal route to the new definition page).
Added nil guard https://github.com/cookpad/barbeque/pull/48/commits/fb056cf9ad29d3b1817d88836b2dbc46c164bc71

File: app/views/barbeque/job_definitions/index.html.haml (hunk 2 above)
Comment (hogelog) [inline app/views/barbeque/job_definitions/index.html.haml]: Sorry I notice `%li.active #{app name}` is `apps#show`.

**Pull request merged**